ansible/roles/common/defaults/main.yaml

```diff
@@ -6,3 +6,13 @@ bin_dir: /usr/bin
 # The directory used by Ansible to temporarily store
 # files on Ansible managed systems.
 ansible_temp_dir: /tmp/.ansible/files
+
+# The directory used for the docker config
+system_config_dir: "/etc/sysconfig"
```

Comment:
Based on your plan ``plan on a follow up PR to move the docker_config_dir to the docker role where it should belong.``, do we need the ``system_config_dir`` variable here at all? ``docker_config_dir`` is already defined in ``docker`` role and used by ``docker`` and ``flannel``.